lib/view_model/send/send_view_model.dart

```diff
@@ -374,7 +374,7 @@ abstract class SendViewModelBase with Store {
     final _priority = priority as TransactionPriority;
     final wallet = _wallet;
 
-    if (isElectrumWallet) {
+    if (hasCoinControl) {
```

Comment:
this is not right, since it uses `bitcoin` instance, separating them into 2 variables would be better as it should stay checked by `isElectrumWallet` or see if this would be also applicable for Monero then let's add it